README.md

```diff
@@ -5,18 +5,27 @@
 It is a fun way to assess your cloud skills. It is also a good representative sample of the work we do at Rearc. Quest is a webapp made with node.js and golang.
 
 ### Q. So what skills should I have ?
-Public cloud (AWS, GCP, Azure). More than one cloud is a "good to have" but one is a "must have". General cloud concepts, especially networking. Docker (containerization). IaC (Infrastructure as code). Linux/Unix. Git. TLS certs is a plus.
+- Public cloud: AWS, GCP, Azure.
+  - More than one cloud is a "good to have" but one is a "must have".
+- General cloud concepts, especially networking.
+- Containerization, such as: Docker, containerd, kubernetes
+- IaC (Infrastructure as code). At least some Terraform preferred.
+- Linux (or other POSIX OS).
+- VCS (Version Control System). Git is highly preferred. 
+- TLS is a plus.
 
 ### Q. What do I have to do ?
 You may do all or some of the following tasks. Please read over the complete list before starting.
 
 1. If you know how to use git, start a git repository (local-only is acceptable) and commit all of your work to it.
-2. Deploy the app in any public cloud and navigate to the index page. Use Linux 64-bit x86/64 as your OS (Amazon Linux preferred in AWS, Similar Linux flavor preferred in GCP and Azure)
-3. Deploy the app in a Docker container. Use `node` as the base image. Version `node:10` or later should work.
-4. Inject an environment variable (`SECRET_WORD`) in the Docker container. The value of `SECRET_WORD` should be the secret word discovered on the index page of the application.
-5. Deploy a load balancer in front of the app.
-6. Use Infrastructure as Code (IaC) to "codify" your deployment. Terraform is ideal, but use whatever you know, e.g. CloudFormation, CDK, Deployment Manager, etc.
-7. Add TLS (https). You may use locally-generated certs.
+2. Deploy the app in any public cloud and navigate to the index page.
```

Comment:
I still wanna change to this:
> Deploy the app in a container in any public cloud -- using whichever services you think best solve this problem -- and navigate to the index page.

Reply:
The following two items address this. This line is redundant so I removed it.